osu.Game/Configuration/SettingSourceAttribute.cs

```diff
@@ -61,12 +63,21 @@ public int CompareTo(SettingSourceAttribute other)
 
     public static class SettingSourceExtensions
     {
-        public static IEnumerable<Drawable> CreateSettingsControls(this object obj)
+        public static IReadOnlyList<Drawable> CreateSettingsControls(this object obj, bool includeDisabled = true) =>
+            createSettingsControls(obj, obj.GetOrderedSettingsSourceProperties(), includeDisabled).ToArray();
+
+        public static IReadOnlyList<Drawable> CreateSettingsControlsFromAllBindables(this object obj, bool includeDisabled = true) =>
```

Comment:
These needs xmldocs. I have no idea what the difference is between the non-all-bindables and the all-bindables ones without reading the code.

Reply:
I'm not sure about keeping this stuff all in the same class... And an "Extensions" class at that. Really, the only thing common between the two is the "controls" part, but the lookup implementation is completely different and has nothing to do with `SettingSource`